### DISCUSSION TITLE
Deleted Repository

### DISCUSSION THREAD

**Discussion by TheFounder90** [community/community#54616] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

When I delete any repository will it also automatically un-publish any pages?

Many thanks

**Comment by hrishikesh-kocharekar-dev** [comment]
When you delete a repository on GitHub, any associated GitHub Pages site will also be removed, and the site will no longer be accessible. Therefore, you don't need to explicitly unpublish your pages before deleting the repository.

It's important to note that once a repository is deleted, it cannot be recovered, including its GitHub Pages site. So if you want to keep a copy of the site's contents, you should make sure to back up the site's files and content before deleting the repository.

**Comment by yoannchaudet** [comment]
Yes, what was said above.

If you want to keep a Pages site around but don't need the repository anymore, you can archive the repository instead.
